### DISCUSSION TITLE
4 column issue in MJML

### DISCUSSION THREAD

**Discussion by sumsho** [mjmlio/mjml#2824] [Q&A] [answered]
Hello! I'm trying to create a 4 column table as the top of a footer. It has our companies logo on the left, and 3 social icons on the right. I'm having issues with the social icons re-sizing as I make the screen smaller. I'm also wondering how I can get the icons right next to each other and remove the space in between. I've pasted my code below, any help welcome (i'm a beginner).

https://mjml.io/try-it-live/lAeOtS8bw7

**Comment by iRyusa** [comment, marked as answer]
You should use mj-social for that inside a column instead.
